i18n/english.yml

```
@@ -489,6 +489,7 @@ components:
       label: Replace %tablePlaceholder% from %versionPlaceholder%
       name: Replace file from version transformation
   FeedTransformationErrors: 
+    csvNameContainsTxt: Custom CSV name cannot contain .txt
```

```suggestion
    csvNameContainsTxt: Custom CSV file name cannot contain .txt
```
Also, do we want to end all validation error messages with a period/full stop?

---

Updated this in https://github.com/ibi-group/datatools-ui/pull/950/commits/51e47ec5e0a32e1ee5752c287a72a9409c9a428b to say "ends with" and "file name". All the validation error messages now have periods.